### ISSUE TITLE
task 2 tan(x) function

### ISSUE THREAD

**Issue by s4il3sh** [ubsuny/23-Homework6G2#55]
How did you guys solve the issue of getting the solution for 90, 270, -90, -270, etc. degrees for the solution of tan(x) function. This solution is not right. How you are dealing with it?

**Comment by tirthbha** [comment]
@s4il3sh I root of tan(X) is pi/2 and that of tanh(x) is 0, so you can chose intervals based on these facts.

**Comment by s4il3sh** [comment]
yes. i chose the interval based on that but the point is, the rootfinding functions throwing out the roots that have infinite values.
